l10n_it_ricevute_bancarie/wizard/wizard_riba_issue.py

```diff
@@ -83,8 +83,8 @@ def create_rdl(
         countme = 1
 
         for move_line in move_lines:
-            if move_line.partner_id.bank_ids:
-                bank_id = move_line.partner_id.bank_ids[0]
+            if move_line.partner_id.sudo().bank_ids:
+                bank_id = move_line.partner_id.sudo().bank_ids[0]
```

Comment:
In chiamata abbiamo detto che sembra non servire

Reply:
perfetto